src/AppBundle/Resources/public/js/app.draw_simulator.js

```diff
@@ -208,6 +208,16 @@ draw_simulator.handle_click = function handle_click(event) {
 		draw_simulator.reset();
 	}
 	var draw;
+	var identityCardCode=app.deck.get_investigator_code()
+	var heroCard = app.data.cards.findById(identityCardCode);
+	var alteregoCard = app.data.cards.findById(heroCard.linked_to_code)
+
+	if(command === 'hero_hand') {
+		draw_simulator.draw(heroCard.hand_size);
```

Comment:
Should this just be setting `draw = heroCard.hand_size` since the `draw` function is still executed at bottom of this function (line 228)?

Reply:
You are right after my new corrections function mentioned by You should be used only once not twice.